.travis.yml

```diff
@@ -29,6 +29,7 @@ env:
   - DIRECTORY="." PLAYBOOK=nightshade-webclients.yml
   - DIRECTORY="." PLAYBOOK=ome-demoserver.yml
   - DIRECTORY="." PLAYBOOK=ome-dundeeomero.yml
+  - DIRECTORY="." PLAYBOOK=training-server.yml
```

Comment:
It looks like `training-server.yml` doesn't have any hardware/low-level config, so is a candidate for a full molecule test instead of just `molecule-syntaxcheck.yml`